src/main/java/co/develhope/spring/controllers/ArticleController.java

```diff
@@ -14,49 +14,47 @@
 
 @RestController
 @RequestMapping("/articles")
-public class ArticlesController {
+public class ArticleController {
 
     @Autowired
-    private ArticlesService articlesService;
+    private ArticleService articleService;
     @Autowired
-    private ArticlesRepository articlesRepository;
-
+    private ArticleRepository articleRepository;
 
     @PostMapping
-    public ResponseEntity<?> createArticle (@Valid @RequestBody Articles articles, BindingResult bindingResult){
+    public ResponseEntity<?> createArticle (@Valid @RequestBody Article article, BindingResult bindingResult){
```

Comment:
c'e' un po di incoerenza tra uso di Entity e DTO per parametri di ingresso e valori in uscita.